learn/safe-core-account-abstraction-sdk/auth-kit.md

```diff
@@ -1,3 +1,93 @@
 # Auth Kit
 
-The Auth Kit uses the [auth-provider](https://github.com/safe-global/account-abstraction-sdk/blob/dev/packages/auth-provider/README.md) to allow users to obtain an Ethereum address using their email or social media account.
+This library provides a way to authenticate blockchain accounts using email addresses, social accounts or traditional web3 wallets (ex. Metamask). When using web2 methods as your email or social account, a derived Ethereum address will be generated.
```

Comment:
```suggestion
The Auth kit uses the [auth-kit SDK](https://github.com/safe-global/account-abstraction-sdk/tree/dev/packages/auth-kit) to create an Ethereum address and authenticate a blockchain account using your email address, social media account, or traditional web3 wallets like Metamask.
```

Reply:
mmm, not sure if any of the descriptions is super clear for me. What do you think about:

```
The [Auth kit](https://github.com/safe-global/account-abstraction-sdk/tree/dev/packages/auth-kit) allows the onboarding of any kind of users (crypto natives or not) by providing several authentication ways as social logins, email accounts or wallets. The result of the authentication is a standard cryptographic key provider for managing the web3 interactions
```

Reply:
What is unclear about it? The description is meant to be a very simple introduction to the kit. We can add more details later like the cryptographic key in subsequent paragraphs.